chaos-workers/chaos-experiments/scripts/verify-readiness.sh

```diff
@@ -8,7 +8,13 @@ namespace=$(getNamespace)
 # verify that everything is running
 
 if [ "${CHAOS_SETUP}" == "cloud" ]; then
-  kubectl wait --for=condition=Ready pod -l app.kubernetes.io/app=zeebe --timeout=900s -n "$namespace"
+  # Wait until brokers are ready. The componenet should be broker, but it is wrongly labelled as gateway
+  kubectl wait --for=condition=Ready pod -l app.kubernetes.io/component=gateway --timeout=900s -n "$namespace"
+  # Wait until all gateway pods are running.
+  kubectl wait --for=condition=Available deployment zeebe-gateway --timeout=120s -n "$namespace"
```

Comment:
It would be nice if we could add a why in the comments here. So why we have to use the wait for pod (broker) ready PLUS deployment avail for gateway.